frontend/src/components/Pages/FrontPage/MissionOverview/SafeZoneBanner.tsx

```diff
@@ -0,0 +1,39 @@
+import { Icon, Typography } from '@equinor/eds-core-react'
+import { tokens } from '@equinor/eds-tokens'
+import { AlertCategory } from 'components/Alerts/AlertsBanner'
+import { useLanguageContext } from 'components/Contexts/LanguageContext'
+import styled from 'styled-components'
+
+const StyledIcon = styled(Icon)`
+    width: 24px;
+    height: 24px;
+    color: ${tokens.colors.text.static_icons__default.hex};
+`
+const StyledLeftContent = styled.div`
+    display: flex;
+    padding-left: 20px;
+    flex-direction: row;
+    align-items: center;
+    gap: 10px;
+`
+
+interface SafeZoneBannerProps {
+    alertCategory: AlertCategory
+}
+
+export const SafeZoneBanner = ({ alertCategory }: SafeZoneBannerProps): JSX.Element => {
+    const { TranslateText } = useLanguageContext()
+
+    return (
+        <>
+            <StyledLeftContent>
+                <StyledIcon name="error_outlined" />
+                <Typography>
+                    {alertCategory === AlertCategory.WARNING
+                        ? TranslateText('Safe zone banner text')
+                        : TranslateText('Dismiss safe zone banner text')}
+                </Typography>
```

Comment:
Figma has 3 different banner. One for driving to safe zone, one for in safe zone and one for leaving safe zone. Should we also have 3 variants?